### DISCUSSION TITLE
SimNIBS GUI loading data error

### DISCUSSION THREAD

**Discussion by gbaghdadi** [simnibs/simnibs#267] [6. Getting started] [answered]
Hi, 
I worked with SimNIBS and SimNIBS example without any issues, but today when I want to use it, I receive the following error after determining the m2m folder. Does anyone know the reason?
"simnibs GUI error operand could not be broadcast together with shape (3,)(0,)"

![image](https://github.com/simnibs/simnibs/assets/40055133/ad5ec5ce-2a49-4e23-afb5-5ae838b049b4)

**Comment by TorgeW** [comment]
Is it possible that you are using a newer numpy version?
`conda list` will show which package versions you are using.

**Reply by vegetablebirdwf** [reply, marked as answer]
Just change the numpy version to a lower version.

**Reply by ncselab** [reply]
Thanks all for your answer. I changed the numpy version from 1.25.2 to a lower version 1.21. but again I receive the same error. Do I need to install a specific version of numpy?
